variantEffectPredictor.wdl

```diff
@@ -403,7 +413,7 @@ task vep {
 
 
     vep --offline --dir ~{vepCacheDir} -i ~{vcfFile} --fasta ~{referenceFasta} --species ~{species} \
-          --assembly ~{ncbiBuild} -o ~{basename}.vep.vcf.gz --vcf --compress_output bgzip ~{addParam} \
+          --assembly ~{ncbiBuild} --custom ~{customTranscriptFile},,gtf -o ~{basename}.vep.vcf.gz --vcf --compress_output bgzip ~{addParam} \
```

Comment:
Is this really two commas? Not a dot?

Reply:
Yes, those two commas specify empty values for options that are not needed  
https://useast.ensembl.org/info/docs/tools/vep/script/vep_custom.html
They show an example:
> # VCF file
>     ./vep [...] --custom https://ftp.ensembl.org/pub/data_files/homo_sapiens/GRCh37/variation_genotype/TOPMED_GRCh37.vcf.gz,,vcf,exact,0,TOPMED

Reply:
Ok, though I only see examples with key/value pairs. I assume, just values in the right place should be supported too. However, the build in Jenkins fails with an error: **vepWorkflow_toMAF_tumor_normal_hg38: Argument missing: variantEffectPredictor.vcf2maf.customTranscriptENSTids** This needs a fix. https://gsici.hpc.oicr.on.ca/job/variantEffectPredictor_vidarr/view/change-requests/job/PR-27/1/console

Reply:
I see, 
I have these two files here: 

> "customTranscriptFile" :"/.mounts/labs/CGI/cap-training/ohamza/VEP/MANE.GRCh38.v1.3.ensembl_genomic.filtered.gtf.gz",
> "customTranscriptENSTids":"/.mounts/labs/CGI/cap-training/ohamza/VEP/MANE.filtered.ENST.ID.txt",

Where do you think I should move them under GSI space ?